e2e/tests/text/adding-text/index.spec.ts

```diff
@@ -80,5 +81,29 @@ for (const splitByGrapheme of [true, false]) {
       name: `5-after-pasting-splitByGrapheme-${splitByGrapheme}.png`,
       maxDiffPixelRatio: 0.03,
     });
+    // go back where we have 2 empty lines, after abc.
+    const clickPointEnd = await textBoxutil.getCanvasCursorPositionAt(20);
+    await canvasUtil.click({
+      position: clickPointEnd,
+      delay: 200,
+    });
+    await page.mouse.down();
+    await page.mouse.move(1, clickPointEnd.y + 150, { steps: 15 });
+    await page.mouse.up();
+    // we remove them because space is finishing
+    await canvasUtil.press('Backspace');
+    // lets click where style end to show that we can add new line without carrying over
```

Comment:
I suggest usign `test.step` to organize the test instead of leaving comments
It is more clear when reading and when looking at test results

Reply:
![screenshot](https://github.com/fabricjs/fabric.js/assets/34343793/2dbe3c8a-6d98-43b5-8b69-893f512ed4ed)
[85800a532aaae57a102d53213c2b8114a0cadb8e.webm](https://github.com/fabricjs/fabric.js/assets/34343793/dfb8a1cc-cd8a-4f35-9db6-db0b9a93557a)

Reply:
![image](https://github.com/fabricjs/fabric.js/assets/34343793/b35c68bc-43f9-4996-9c7d-636084458da6)